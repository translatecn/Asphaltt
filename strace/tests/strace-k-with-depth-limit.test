#!/bin/sh
#
# Check strace -k --stack-trace-frame-limit for attached tracees.
#
# Copyright (c) 2020-2024 The strace developers.
# All rights reserved.
#
# SPDX-License-Identifier: GPL-2.0-or-later

KOPT_SHORT=-k
KOPT_LONG=--stack-trace
KOPT_EXTRA="--stack-trace-frame-limit 3"

. "${srcdir=.}"/strace-k.test
