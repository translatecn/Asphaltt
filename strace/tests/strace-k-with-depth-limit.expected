^chdir ((__)?chdir f3 f2|__kernel_vsyscall (__)?chdir f3) _too_many_stack_frames_
^SIGURG ((__)?chdir f3 f2|__kernel_vsyscall (__)?chdir f3) _too_many_stack_frames_
